src/content/fr/2019/javascript.md

```diff
@@ -0,0 +1,433 @@
+
+Envoyer de plus petits paquets de JavaScript au navigateur est le meilleur moyen de réduire les temps de téléchargement et d’améliorer ainsi les performances des pages. Mais quelle quantité de JavaScript utilisons-nous réellement&nbsp;?
+
+<figure>
+   <a href="/static/images/2019/javascript/fig1.png">
+      <img src="/static/images/2019/javascript/fig1.png" alt="Figure 1. Répartition des octets de JavaScript par page." aria-labelledby="fig1-caption" aria-describedby="fig1-description" width="600" height="371" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-iframe="https://docs.google.com/spreadsheets/d/e/2PACX-1vTpzDb9HGbdVvin6YPTOmw11qBVGGysltxmH545fUfnqIThAq878F_b-KxUo65IuXaeFVSnlmJ5K1Dm/pubchart?oid=1974602890&format=interactive">
+   </a>
+   <div id="fig1-description" class="visually-hidden">Diagramme à barres montrant que 70 octets de JavaScript sont utilisés dans le percentile p10, 174 octets pour p25, 373 octets pour p50, 693 octets pour p75 et 1093 octets pour p90</div>
+   <figcaption id="fig1-caption">Figure 1. Répartition des octets de JavaScript par page.</figcaption>
+</figure>
+
+La Figure 1 ci-dessus montre que nous utilisons 373&nbsp;Ko de JavaScript au 50ᵉ percentile (aussi appelé médiane). En d’autres termes, 50&nbsp;% des sites envoient plus que cette quantité de JavaScript à leurs utilisateurs.
+
+En regardant ces chiffres, il est naturel de se demander si ce n’est pas trop de JavaScript. Cependant, en termes de performances des pages, l’impact dépend entièrement des connexions réseau et des appareils utilisés. Ce qui nous amène à notre prochaine question&nbsp;: quelle quantité de JavaScript envoyons-nous lorsque nous comparons les tests mobiles et sur ordinateurs de bureau&nbsp;?
+
+<figure>
+   <a href="/static/images/2019/javascript/fig2.png">
+      <img src="/static/images/2019/javascript/fig2.png" alt="Figure 2. Distribution de JavaScript par page et par type d’appareil." aria-labelledby="fig2-caption" aria-describedby="fig2-description" width="600" height="371" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-iframe="https://docs.google.com/spreadsheets/d/e/2PACX-1vTpzDb9HGbdVvin6YPTOmw11qBVGGysltxmH545fUfnqIThAq878F_b-KxUo65IuXaeFVSnlmJ5K1Dm/pubchart?oid=1914565673&format=interactive">
+   </a>
+   <div id="fig2-description" class="visually-hidden">Un diagramme à barres montrant que 76 octets&#8239;/&#8239;65 octets de JavaScript sont utilisés dans le percentile p10 sur ordinateur de bureau et mobile respectivement, 186&#8239;/&#8239;164 octets pour p25, 391&#8239;/&#8239;359 octets pour p50, 721&#8239;/&#8239;668 octets pour p75, et 1&#8239;/&#8239;131&#8239;/&#8239;1&#8239;/&#8239;060 octets pour p90.</div>
```

Comment:
Is it a bit OTT to use `&#8239` so much?

Reply:
Is it ?